Certification-Standard/introduction.md

```diff
@@ -0,0 +1,239 @@
+
+## 3.2.3 Parametrization of a TOE
+
+A DApp could be parametrized in many ways prior to deployment. Parts of a DApp could be created on-the-fly by the DApp itself. The TOE should clearly list the set of parameters that were considered.
+
+An evaluation of such a TOE should clearly list the set of parameters that were considered and on which the evaluation was based on. A certificate can only refers to a version of the TOE with a parameter choice that matches one in the set considered for the evaluation.
+
+## 3.2.4 Operational environment of the TOE
+
+Everything outside the TOE boundary belongs the Operational Environment (OE) of the TOE. In the case of the evaluation of part of DApp, the non-TOE parts of the DApp are parts of the OE.
+
+The ST needs to describe the assumptions on the OE so that together with the security functionalities provided by the TOE, the security objectives are met.
+
+## 3.3 Specifying Security Requirements
+
+### 3.3.1 Security Problem Definition
+
+The SPD defines the security problem that is to be adressed and is presented in the ST. The SPD defines the threats that are to be countered by the TOE, the OE assumptions, and the organizational security policies.
+
+#### Threats
+
+The SPD describes the threats that the TOE is intended to counter.
+
+A threat is an adverse action that can be performed by a threat agent on an asset.
+
+## 3.4 Security Target
+
+A ST is a document that describes:
+- a specific TOE (features, life cycle, interfaces, the intended usage, ...)
+- Conformance Claims (Which version of the standard, which PP, which EAL, ...)
+- Security problem definition (assets, threats, organizational security policies, assumptions, ...)
+- Security objectives (functional and assurance)
+- Security requirements (functional and assurance)
+
+
+## 3.5 Protection Profile
+
+*RSoulat* I think we can drop the PP for a first version. It might become clearer when we have first drafts for example STs to ground the discussion and see which PP we can establish.
+
+## 3.5 Evaluation Levels
+
+The CCS should define a set of evaluation levels that can be used to evaluate the security features of a DApp. The evaluation levels should be defined in a way that they can be used to compare the security features of different DApps.
+
+### 3.5.1 EAL1 - Automated testing
+
+This level of evaluation is based on the functional testing of the TOE to determine whether the TOE meets the security requirements. The testing is performed by the developer of the TOE and the use of automated tools.
+
+At this level, the TOE is not expected to be resistant to attack. The TOE is expected to be functionally correct and to be free of vulnerabilities that are checkable automatically.
+
+At this level of evaluation, the analysis of the Security Target has to be left to the end-user. The quality of the Security Target cannot be addressed automatically.
```

Comment:
Are evaluators not involved at this level at all?

Reply:
That's an open question I believe. Do we require that no matter what, an evaluator has to do the certification or should an automated system be able to do it?